code/modules/materials/materials.dm

```diff
@@ -681,7 +681,8 @@ var/list/name_to_material
 	name = MATERIAL_SILK
 	stack_origin_tech = list(TECH_MATERIAL = 2)
 	stack_type = /obj/item/stack/material/silk
-	composite_material = list(MATERIAL_BIOMATTER = 1) //So we have a vaule to more then one faction
```

Comment:
is this intented to be removed

Reply:
Fixed in the current iteration, lemme push.